source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+
+1. Store your `.env` outside of the project root on the Pantheon server. 
+
+    - Bedrock is configured to look for the `.env` file in the `files/private` directory by default. This is also the location where Pantheon stores all files that shouldn't be accessible to the public.
+
+1. Commit the `vendor` and `web` directories.
+
+    - Pantheon does not support using Composer to install dependencies that are deployed to Pantheon servers. You must commit the entire `vendor` and `web` directories into your Git repo to ensure that your dependencies are installed for the new project.
+
+1. Convert the `web/app/uploads` directory to a [symlink](https://github.com/git-for-windows/git/wiki/Symbolic-Links) that points to the `files` directory. 
+
+    - This is necessary because Pantheon stores media files in the `files` directory instead of the `uploads` directory.
+
+1. Create the initial `.env` file.
+
+    - This is necessary because Bedrock uses environment variables to manage sensitive credentials. These environment variables usually come from multiple sources. The most important variable is the `.env` file which Bedrock uses instead of the standard WordPress `wp-config.php` file. Pantheon was not designed to create the initial `.env` file that your Bedrock site requires. You must create it yourself and upload it to the Pantheon server. The easiest way to do that is by connecting to your Pantheon server using FTP and creating the `.env` file. 
+
+1. Upload the newly created `.env` file by connecting to the Pantheon server using an FTP.
+
+    - To find your connection information: Navigate to the site's **Dev** environment in Pantheon > click **Connection Info** > scroll to the bottom of the menu to view your SFTP connection information.
+
+1. Connect to your Pantheon site using the SFTP credentials located in the previous step > navigate to the `files` directory > create a `private` directory > create the `.env` file in the `private` directory by running the following command:
+
+    ```bash
+    WP_ENV=development
+    WP_SITEURL=${WP_HOME}/wp
+
+    Generate your keys here: https://roots.io/salts.html
+    AUTH_KEY='generateme'
+    SECURE_AUTH_KEY='generateme'
+    LOGGED_IN_KEY='generateme'
+    NONCE_KEY='generateme'
+    AUTH_SALT='generateme'
+    SECURE_AUTH_SALT='generateme'
+    LOGGED_IN_SALT='generateme'
+    NONCE_SALT='generateme'
+    ```
+
+    - Your `.env` will be created. Your new `.env` file is smaller than the original Bedrock `.env` file because Pantheon automatically supplies many of the environment variables that are usually stored in the `.env` file.
+
+1. Replace all placeholder keys with the [new keys](https://roots.io/salts.html).
+
+1. Use [SFTP or git](/guides/wordpress-git/) to push changes directly to your Pantheon site.
+
+## See Also
+
+- [Pantheon WordPress with Bedrock Recommended GitHub Repository](https://github.com/pantheon-systems/wordpress-bedrock-recommended)
```

Comment:
This is an unsupported prototype repository that probably we shouldn't link out to until it's in a more finalized state.